script/Arcueid/Day 8/08_00_ARC08_6B.txt

```diff
@@ -576,7 +576,8 @@ Sensing the abnormality, Arcueid turns around.
 [sha:fdc9af8d3d6ee171f7ce3a03c5c7304356dd7792]{
 -- Page 391, Offset 20661.
 -- 振るわれた触手は鞭のように伸縮し、アルクェイドの首筋に振り落とされた。
-The wielded tentacle releases like a whip and aims toward Arcueid's neck.
+// jesus take the fucking wheel i dont undersstand this shit
```

Comment:
this is kinda simplified but I think its got all of the essentials in there
>振るわれた触手は鞭のように伸縮し、アルクェイドの首筋に振り落とされた。
Like a whip, a tentacle shoots out straight for the nape of Arcueid's neck.